blog/2021-09-08-CWA-2-9/index.md

```diff
@@ -58,7 +58,7 @@ Users who get vaccinated with **Johnson & Johnson’s vaccine** after recovery a
 Furthermore, users who have received a **booster vaccination** can transfer their digital certificate into the Corona-Warn-App. Booster vaccinations are displayed like the other vaccination certificates with "Vaccination n of m".
 
 <br></br>
-<center> <img src="./booster-impfung_v2_en.png" title="Auffrischungsimpfung" style="align: center" width=250> </center>
+<center> <img src="./booster-impfung_v2_en.png" title="Auffrischimpfung" style="align: center" width=250> </center>

```

Comment:
```suggestion
<center> <img src="./booster-impfung_v2_en.png" title="Booster Vaccination" style="align: center" width=250> </center>
```

The original text was in the wrong language.

Reply:
@MikeMcC399 Thanks.